examples/connect/src/ExampleComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import {Composite, ProgressBar, Properties, Stack, TextView} from 'tabris';
+import {connect, prop} from 'tabris-decorators';
+
+@connect(state => ({
+  selection: state.num,
```

Comment:
That's a pattern/flaw of most of these very technical examples. They are deliberately not real-world-like to keep them simple and on point, but it can also make them less expressive. What should I use instead? foo/bar? age/name?

Reply:
I will update this the next time i touch the examples anyway.